data/ontology.yaml

```diff
@@ -0,0 +1,30 @@
+# Define the ontology that is available for Timesketch.
+# An ontology needs to define the name, which is what will
+# be used to identify the ontology.
+# + cast_as is used to define how the value should be
+#       interpreted in code.
+# + description is a short text describing the ontology.
+
+text:
+        cast_as: str
+        description: "Free flowing text."
+
+int:
+        case_as: int
+        description: "Integer"
+
+url.safe:
+        cast_as: str
+        description: "Safe URL that can be visited."
```

Comment:
Hmm, do we need this? Are URLs ever going to be clickable?

Reply:
yes, they might be.... imagine I add a link to an investigation, such as a link to my SIEM solution, or a link to a threat report, or references to a document containing further information....

Reply:
this would mean that the UI would render that attribute as a href, so it would be clickable and would open up a page/site

Reply:
ack